src/simple/Phone.js

```diff
@@ -105,6 +114,14 @@ class Phone extends Emitter {
     return this.phone.sendMessage(toSipSession, body);
   }
 
+  sendChat(content: string) {
```

Comment:
Can you add an optional `sipSession` parameter like in `sendMessage` ?
 It will the ease to understand that we send a chat to a callSession.